src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: fit-content;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
```

Comment:
The selector `&--4 &__star:nth-child(-n + 4)` might not work as intended if the `&--4` modifier is not correctly applied to the `.stars` class. Ensure that the modifier class is applied to the `.stars` element in the HTML to activate the first four stars.